packages/api-client/src/components/Sidebar/SidebarListElement.vue

```diff
@@ -27,11 +28,18 @@ const emit = defineEmits<{
 
 const router = useRouter()
 
-const handleNavigation = (event: MouseEvent, uid: string) => {
+const handleNavigation = (
+  event: MouseEvent,
+  uid: string,
+  collectionId?: string,
+) => {
+  const path = collectionId
+    ? `/workspace/default/environment/${collectionId}/${uid}`
+    : `/workspace/default/environment/${uid}`
   if (event.metaKey) {
-    window.open(uid, '_blank')
+    window.open(path, '_blank')
```

Comment:
For the future but should be make this open a tab on the desktop client?

Maybe we should handle window.open for the desktop client?

Reply:
+10